bot/cogs/base_converter_cog.py

```diff
@@ -0,0 +1,82 @@
+import discord
+import discord.ext.commands as commands
+
+from bot.consts import Colors
+
+class BaseConverterCog(commands.Cog):
+
+    def __init__(self, bot) -> None:
+        self.bot = bot
+
+    @commands.group(pass_context= True, invoke_without_command= True, aliases= ['convert','baseconvert'])
+    async def bconvert(self, ctx, *, number) -> None:
+        """
+        A simple base converter that takes in a base number an a value, then displays the value in binary, octal, decimal, and hexadecimal
+
+        Format: bconvert [base abbreviation OR full base name] [number OR number with base prefix]
+
+                base abbreviations: [bin / dec / hex / oct]
+                full base name: [binary / decimal / hexadecimal / oct]
+
+                number examples: [11, 99, ff, 77 ]
+                number with base prefix examples: [0b11, 99, 0xff, 0o77]
+        """
+        if ctx.invoked_command is None:
+            await ctx.send('Invalid entry! Use the help command for examples!')
+        return
```

Comment:
we dont need this at all, this will throw a NullRefException if someone does` !convert`

The bconvert method should list out the possible formats of the commands. You dont need to handle the invoked_command stuff cuz d.py does all that for you

Reply:
Oh! That's nice. Thanks